Apps/Examples/Examples/All Examples/NavigationSimulatorExample.swift

```diff
@@ -41,7 +41,8 @@ final class NavigationSimulatorExample: UIViewController, ExampleProtocol {
     private func configureMap() {
         navigationSimulator = NavigationSimulator(viewport: mapView.viewport, route: sampleRouteLine)
 
-        mapView.location.options.puckType = .puck2D(.makeDefault(showBearing: true))
+        let configuration = Puck2DConfiguration(topImage: UIImage(named: "user_puck_icon")!)
+        mapView.location.options.puckType = .puck2D(configuration)
```

Comment:
Double-check that this change was intentional

Reply:
Yes: to match the Android puck